src/cogs/forums.py

```diff
@@ -93,7 +98,7 @@ async def to_msg_data(message):
 
         data = list(itertools.chain.from_iterable(await asyncio.gather(*[to_msg_data(msg) for msg in msgs])))
 
-        thread, _ = await channel.create_thread(name = f"{msg.author.display_name}'s issue", **data.pop(0))
+        thread, _ = await channel.create_thread(name = f"{msg.author.display_name}'s issue", content = f"The following messages were originally posted by <@{msg.author.id}> in <#{ctx.channel.id}> but were moved into a help thread here to keep order:")
```

Comment:
` f"The following messages were originally posted by <@{msg.author.id}> in <#{ctx.channel.id}>:"`
is sufficient

Reply:
`but were moved into a help thread here to keep order` 
that's implied. Please remove it, bot messages should be concise.

Reply:
I already changed this in the latest commit.